- added rkh framework 3.2.3

fbucafusco opened the pull request:
- added rkh blinky demo for edu_ciaa_nxp

fbucafusco commented:
hice el PR de 242e4a1
luego por mi lado hice otro commit, y automaticamente se agrego al PR.... 
se puede evitar esto?

martinribelotta commented:
Por cada commit que haces, si no cerras el PR te lo deja automáticamente vinculado, justamente para que no tengas que hacer un PR nuevo cada vez.
By the way... el Makefile y el README te quedaron con fin de linea Unix!!! no es un problema per se, pero rompe completamente el track de cambios (de un momento para otro, lo que vez en el historial de git es que se cambio TODO el archivo de pies a cabeza).
Si podes, pasalo a fin de linea unix (y hacele eso a todo el texto que tengas para unificar de paso)

Yo, para eso, uso el comando:

```sh
find . -type f -not -ipath './.git/*' -exec file "{}" \;|grep CRLF|cut -f 1 -d ':'|xargs dos2unix -u
```

Que anda con las herramientas de busybox incluidas en embedded-ide

fbucafusco commented:
Si puede ser.  A veces edito en windows y otras en "linux" (wsl). Y cada uno aporta si granito .jaja. 
Bueno. Corro esa linea y listo! No problem.

fbucafusco committed:
- added rkh framework 3.2.3
- added rkh blinky demo for edu_ciaa_nxp
- added rkh shared example [does not work yet, but compiles]

fbucafusco commented:
quedaron bien ahi? 
me vuelve loco esto. Parece que git tambien los cambia autimaticamente.... segun una configuracion que no logro cambiar. jaja.

epernia commented:
Muchas gracias Franco!!!
